src/tests/Eip1993Factories.test.ts

```diff
@@ -0,0 +1,38 @@
+import {HTTPProvider} from '@zilliqa-js/core';
+import {Eip1993Factories} from '../utils/Eip1993Factories';
+import nock from 'nock';
+
+beforeEach(() => {
+  nock.enableNetConnect();
+  jest.restoreAllMocks();
+});
+
+describe('Eip1993Factories', () => {
+  describe('fromZilliqaProvider', () => {
+    it('should create provider from zilliqa provider', async () => {
```

Comment:
Could you add tests for Resolution class configured to use converted Zilliqa provider?